Packages/com.unity.inputsystem/Documentation~/select-control-binding.md

```diff
@@ -0,0 +1,52 @@
+# Pick a control for a binding
```

Comment:
The title of this page is different from what's in the TOC

Reply:
addressed in new changes